### ISSUE TITLE
Add shellcheck

### ISSUE THREAD

**Issue by ju5tevg3niy** [clearlinux/distribution#2023]
Official package name: shellcheck

License (must be an OSI approved Open Source license): GPL-3.0

Download URL of latest release: https://github.com/koalaman/shellcheck/releases

Latest release date (must be recent): 5 Apr 2020

Description: 
Static analysis tool for shell scripts. I find it useful because bash scripts are essential tool in the Linux ecosystem. Bash scripts are powerful, but fragile and many users may want to check them with a simple `shellcheck some_script.sh`. It is also used by various tools and IDEs.
https://www.shellcheck.net/

**Comment by mdhorn** [comment]
This is a duplicate of a long-standing request, #271 
Looks like it is still blocked on the prerequisite build tool of [cabal](https://www.haskell.org/cabal/).
